lib/matplotlib/artist.py

```diff
@@ -106,6 +131,8 @@ def __init_subclass__(cls):
         # Inject custom set() methods into the subclass with signature and
         # docstring based on the subclasses' properties.
 
+        cls.draw = _prevent_rasterization(cls.draw)
```

Comment:
Either the above comment needs updating, or this should not go between it and the code below.

Reply:
Thanks. I will move this line above the comment.

Reply:
We should be more careful here about not not over-writing the `draw` method unless we have to, I will push a commit to try and fix this.